ansible/roles/containers/tasks/atfncat.yaml

```diff
@@ -0,0 +1,59 @@
+- name: Create atfncat container
+  vars:
+    psrcat_url: "{{ item.psrcat_url }}"
+    psrcat_version_number: "{{ item.psrcat_version_number }}"
+    psrcat_checksum: "{{ item.psrcat_checksum_sha256 }}"
```

Comment:
I do the same in the RStudio container (because I think it does R and RStudio separately), and probably elsewhere too. But for future containers the `psrcat` prefix is _slightly_ redundant. But mainly I don't care.